src/app/compressed-air-assessment/system-profile-graphs/system-profile-graphs.component.ts

```diff
@@ -346,10 +348,10 @@ export class SystemProfileGraphsComponent implements OnInit {
         }
         traceData.push(trace);
       });
-      let xRangeMax: number = this.profileSummary[0].profileSummaryData.length > 1 ? 24 : 1;
-      var layout = this.getLayout("Compressor Capacity (%)", [1, xRangeMax], [0, 105], '%');
+      let xRangeMax: number = this.profileSummary[0].profileSummaryData.length > 1 ? 25 : 1;
```

Comment:
We shouldn't change any of the range values for the charts unless approved by Kristina/CA team. I think we might have tried to add a padded axis like this when it was created, and they preferred it without.

Another option: in non-print view display the charts with the original axis. When in print view assign the print view values from an object.